Jint/Native/WeakMap/WeakMapPrototype.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Jint.Collections;
+using Jint.Native.Object;
+using Jint.Native.Symbol;
+using Jint.Runtime;
+using Jint.Runtime.Descriptors;
+using Jint.Runtime.Interop;
+
+namespace Jint.Native.WeakMap
+{
+    /// <summary>
+    /// https://262.ecma-international.org/11.0/#sec-weakmap-objects
```

Comment:
nit: the living spec links can be more useful as they are not tied to specific version https://tc39.es/ecma262/#sec-weakmap-objects